pom.xml

```
@@ -186,5 +186,10 @@
             <artifactId>commons-lang3</artifactId>
             <version>3.12.0</version>
         </dependency>
+        <dependency>
+            <groupId>co.zecko.common.java</groupId>
+            <artifactId>client</artifactId>
```

```suggestion
            <artifactId>public</artifactId>
```

The private one can be: `co.zecko.common` and `private`